transcom/mymove
# Int b 21457 too approve email for retiree separatee

## [Agility ticket](tbd)

## Summary
Currently when a TOO creates a move task order , emails that gets sent to retirees and separatees state that their move is scheduled to deliver to their HOS/HOR/PLEAD instead of the delivery address on their shipment. So these changes are to let the delivery address on the email to display the right address.

[this article](tbd) explains more about the approach used.

## Verification Steps for the Author

These are to be checked by the author.


## Verification Steps for Reviewers

These are to be checked by a reviewer.


### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/getting-started/application-setup/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/getting-started/development/testing)
**Local testing for emails, you will need to start the server with this:**
EMAIL_BACKEND=ses AWS_REGION=us-gov-west-1 aws-vault exec transcom-gov-dev -- make server_run

### How to test

1. Access the application
2. Login as a Customer
3. Create a new move (Select Retirement/ Separation) on the "Tell us about your move orders page".
4. Go through the process of Setting up a shipments (e.g HHG/PPM) to create a shipment , sign a complete(Make sure to copy/save your move code for later).
5. Login as a Service Counselor(SC)
6. Search for your move code in step 4 and navigate to it.
7. Make sure you edit/fix any required fields.
8. Login as a TOO
9. Repeat step 6
10. Select the checkbox  for the shipment created in 4
11. Scroll down and click "Approve selected" button.
12. Click "Approve and send" button.
13. You will receive an email (**it took ~ 15 mins during testing**) with subject"Your personal property move has been ordered" in the email you provided during the Move/shipment creation process.
14. Note that the verbiage in the email "This is a confirmation that a move task order has been placed for your move  [YOUR_MOVE_CODE] to [Destination_address_you_provided_for_shipment].

### Frontend


### Backend


### Database

#### Any new migrations/schema changes:


## Screenshots
[B-21457-TOO-Approved-Move-Details-Emails.docx](https://github.com/user-attachments/files/17560843/B-21457-TOO-Approved-Move-Details-Emails.docx)
